### PR TITLE
Projact 1.4 developerguide

### PR DESCRIPTION
**Pull request by akgrenSoar**
Minor sentence changes to Developer Guide
Edited StorageClassDiagram
Edited ModelClassDiagram

### CODE REVIEW

**Review by siqiang-ng — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -443,17 +443,17 @@ The diagram below shows a sample interaction of `TaskDoneCommand`.
 * can type fast
 * is reasonably comfortable using CLI apps
 * wants to group contacts based on the different modules
-* has a need to store the different meeting platforms for each module project in one platform
-* wants a task tracker
+* has a need to store different url links for each module in a single accessible location
+* wants to keep track of tasks and deadlines
```
Comment (khinkhinn) [inline docs/DeveloperGuide.md]: we cant keep track of deadlines right?

Comment (siqiang-ng) [reply docs/DeveloperGuide.md]: i guess deadlines in the form of words?

Comment (akgrenSoar) [reply docs/DeveloperGuide.md]: resolved. Removed the word 'deadline'

**Pull request merged**